src/renderer/material.rs

```diff
@@ -158,6 +162,8 @@ pub struct FragmentAttributes {
     pub uv: bool,
     /// Color: `in vec4 col;`
     pub color: bool,
+    /// Instance ID: `in uint instanceID`
```

Comment:
This is not true 🤔

Reply:
Leftover evidence of when I tried to pass it as a `uint`.